modules/migration-kubernetes-objects.adoc

```diff
@@ -44,3 +50,19 @@ spec:
 ----
 <1> Specify the Kubernetes object, for example, `Secret` or `ConfigMap`.
 <2> Specify the label of the resources to migrate, for example, `app: frontend`.
+
```

Comment:
@RichardHoch 
Are you sure that both `includedResources` and `labelSelector` should appear in the `spec` section? This seems to contradict your updated explanation of the options for adding Kubernetes objects. Aren't they mutually exclusive?

Reply:
@sbeskin-redhat They are not mutually exclusive. You can add `labelSelector` alone which will match "all" resources that match the selector. The expected use case is to select a subset of resources (not all) via `includedResources` and then apply an additional `labelSelector` to filter within those resources.